README.md

```diff
@@ -61,17 +61,12 @@ cd $(find . -type d | pick)      # Select a directory to cd into
 Pick can also easily be used from within Vim with this VimL function:
 
 ```viml
-function! PickCommand(choice_command, pick_args, vim_command)
-  try
-    let selection = system(a:choice_command . " | ./src/pick" . a:pick_args)
-  catch /Vim:Interrupt/
```

Comment:
I actually do not know. My theory is that selecta handles control characters differently.